lib/travis/build/script/shared/jdk.rb

```diff
@@ -44,6 +68,38 @@ def uses_java?
           def uses_jdk?
             !!config[:jdk]
           end
+
+          def jdk
+            config[:jdk].gsub(/\s/,'')
+          end
+
+          def use_install_jdk?(jdk)
+            ! install_jdk_args(jdk).empty?
+          end
+
+          def download_install_jdk
+            return if app_host.empty?
+            sh.cmd "curl -sf -O https://#{app_host}/files/install-jdk.sh"
+          end
+
+          def install_jdk_args(jdk)
+            args_for = {
+              # OpenJDK
+              'openjdk-ea'   => '-L GPL',
+              'openjdk9'     => '-F 9  -L GPL',
+              'openjdk10'    => '-F 10 -L GPL',
+              'openjdk11'    => '-F 11 -L GPL',
```

Comment:
You may want to use `-C` for all openjdk builds - https://github.com/sormuras/bach/pull/12

Reply:
Thought about that too. But it would change the "default" setup.

Users just configuring `jdk:openjdkX` on Travis CI will be surprised if their local openjdkX installation behaves differently.

Documenting the `-C` switch when "openjdk" is selected is a good thing. Like was done with the cache configuration warning.